packages/flutter/test/material/dialog_test.dart

```diff
@@ -563,7 +563,74 @@ void main() {
     final Rect buttonOneRect = tester.getRect(find.byKey(key1));
     final Rect buttonTwoRect = tester.getRect(find.byKey(key2));
     // Second [RaisedButton] should appear above the first.
-    expect(buttonTwoRect.bottom, buttonOneRect.top);
+    expect(buttonTwoRect.bottom, lessThanOrEqualTo(buttonOneRect.top));
+  });
+
+  testWidgets('Dialogs have no spacing by default for overflowing actions', (WidgetTester tester) async {
+    final GlobalKey key1 = GlobalKey();
+    final GlobalKey key2 = GlobalKey();
+
+    final AlertDialog dialog = AlertDialog(
+      title: const Text('title'),
+      content: const Text('content'),
+      actions: <Widget>[
+        RaisedButton(
+          key: key1,
+          onPressed: () {},
+          child: const Text('Looooooooooooooong button 1'),
+        ),
+        RaisedButton(
+          key: key2,
+          onPressed: () {},
+          child: const Text('Looooooooooooooong button 2'),
+        ),
+      ],
+    );
+
+    await tester.pumpWidget(
+      _appWithAlertDialog(tester, dialog),
```

Comment:
@shihaohong looks like this line broke the build.

Reply:
https://cirrus-ci.com/task/5286501055463424

Reply:
I think another PR was merged that changed `_appWithAlertDialog`'s function signature recently. Should be an easy fix!